docs/source/markdown/options/uidmap.container.md

```diff
@@ -76,8 +84,111 @@ Every additional range is added sequentially afterward:
 | 1                     | $FIRST_RANGE_ID         | $FIRST_RANGE_LENGTH |
 | 1+$FIRST_RANGE_LENGTH | $SECOND_RANGE_ID        | $SECOND_RANGE_LENGTH|
 
+  `Referencing a host ID from the parent namespace`
+
+As a rootless user, the given host ID in **--uidmap** or **--gidmap**
+is mapped from the *intermediate namespace* generated by podman. Sometimes
+it is desirable to refer directly at the *host namespace*. It is possible
+to manually do so, by running `podman unshare cat /proc/self/gid_map`,
+finding the desired host id at the second column of the output, and getting
+the corresponding intermediate id from the first column.
+
+Podman can perform all that by preceding the host id in the mapping
+with the `@` symbol. For instance, by specifying `--gidmap 100000:@2000:1`,
+podman will look up the intermediate id corresponding to host id `2000` and
+it will map the found intermediate id to the container id `100000`. The
+given host id must have been subordinated (otherwise it would not be mapped
+into the intermediate space in the first place), and the `amount` of any
+mapping using the `@` symbol must be of size 1, because consecutive host
+id mappings may not be consecutive mappings at the intermediate id
+namespace.
```

Comment:
I will implement support for size > 1 following this description:

>  `Referencing a host ID from the parent namespace`
>
>As a rootless user, the given host ID in **--uidmap** or **--gidmap**
>is mapped from the *intermediate namespace* generated by podman. Sometimes
>it is desirable to refer directly at the *host namespace*. It is possible
>to manually do so, by running `podman unshare cat /proc/self/gid_map`,
>finding the desired host id at the second column of the output, and getting
>the corresponding intermediate id from the first column.
>
>Podman can perform all that by preceding the host id in the mapping
>with the `@` symbol. For instance, by specifying `--gidmap 100000:@2000:1`,
>podman will look up the intermediate id corresponding to host id `2000` and
>it will map the found intermediate id to the container id `100000`. The
>given host id must have been subordinated (otherwise it would not be mapped
>into the intermediate space in the first place).
>
>**If the length is greater than one, for instance with `--gidmap 100000:@2000:2`,
>podman will map host ids `2000` and `2001` to `100000` and `100001` respectively,
>regardless of how the intermediate mapping is defined.**

I think this is what any user would expect. With this notation, the user may not even be
aware that an intermediate mapping exists